CHANGELOG.md

```diff
@@ -50,6 +50,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+- (eibc) [#845](https://github.com/dymensionxyz/dymension/issues/845) Segregate eibc events as per use cases
```

Comment:
I remember Omri said we don't need manually update `CHANGELOG.md` since a bot will handle that